core/src/main/java/org/mifos/mobile/core/ui/component/AboutUsItemCard.kt

```diff
@@ -0,0 +1,42 @@
+package org.mifos.mobile.core.ui.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun AboutUsItemCard(title: String, subtitle: Int?, iconUrl: Int?) {
+    Row(
+        modifier = Modifier.padding(16.dp)
+    ) {
+        iconUrl?.let { painterResource(id = it) }?.let {
+            Image(
+                painter = it,
+                contentDescription = null,
+                modifier = Modifier.padding(0.dp, 0.dp, 10.dp, 0.dp)
+            )
+        }
+        Column {
+            Text(
+                text = title,
+                style = MaterialTheme.typography.bodyLarge,
+                modifier = Modifier.padding(0.dp, 0.dp, 10.dp, 0.dp)
+            )
+            if (subtitle != null) {
+                Text(
+                    text = stringResource(id = subtitle),
+                    style = MaterialTheme.typography.bodyLarge,
+                    modifier = Modifier.padding(0.dp, 0.dp, 10.dp, 0.dp)
```

Comment:
For spacing and paddings use material design recommended values which are (8, 12, 16, 24)

Reply:
I had to skip this review as the rows had started to look bloated. I went with the values of 8.dp instead.